web/src/server_events_dispatch.js

```diff
@@ -777,12 +777,14 @@ export function dispatch_normal_event(event) {
                 $("body").toggleClass("less-dense-mode");
                 $("body").toggleClass("more-dense-mode");
                 information_density.set_base_typography_css_variables();
+                information_density.calculate_timestamp_widths();
```

Comment:
I don't know whether it makes sense to create a wrapper function like `update_information_density()`, so I've erred on the side of verbosity here and in the font-size/line-height events below.

Reply:
This seems OK for now, we can always refactor as we get more of a feeling for how much stuff needs to be here.